src/caselawclient/models/utilities/aws.py

```diff
@@ -146,3 +151,40 @@ def notify_changed(uri: str, status: str, enrich: bool = False) -> None:
         Subject=f"Updated: {uri} {status}",
         MessageAttributes=message_attributes,
     )
+
+
+def copy_assets(old_uri: str, new_uri: str) -> None:
```

Comment:
```suggestion
def copy_unpublished_assets(old_uri: str, new_uri: str) -> None:
```

rename based off the docstring which says `Copy *unpublished* assets...`